src/modules/personenkontext/api/created-personenkontext-organisation.dto.ts

```diff
@@ -1,3 +1,14 @@
+import { AutoMap } from '@automapper/classes';
+
 export class CreatedPersonenkontextOrganisationDto {
-    public readonly id!: string;
+    @AutoMap(() => String)
+    public id!: string;
```

Comment:
also no need AutoMap

Reply:
This class is used in the `person-api.mapper.profile.ts`, so I would leave this annotation for now and remove it, when it's not used in any mapper profile.